main.css

```diff
@@ -7,6 +7,79 @@ html, body
     flex-direction: column;
 }
 
+footer
```

Comment:
This looks like code from our old footer? This regresses back to when then CSC copyright was not in the sidebar and we couldn't click the Leaflet link. Make sure you pull those recent changes